elide-core/src/main/java/com/yahoo/elide/core/filter/strategy/DefaultFilterStrategy.java

```diff
@@ -0,0 +1,195 @@
+/*
+ * Copyright 2016, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.core.filter.strategy;
+
+import com.yahoo.elide.core.EntityDictionary;
+import com.yahoo.elide.core.filter.Operator;
+import com.yahoo.elide.core.filter.Predicate;
+import com.yahoo.elide.core.filter.expression.AndExpression;
+import com.yahoo.elide.core.filter.expression.Expression;
```

Comment:
`Expression` feels to generic. As I read through the code, I keep finding myself having to recall what namespace this `Expression` is under. Should we rename this to something like `FilterExpression`? Similarly, for Elide 3.0 we should consider this update for Permission `Expression`'s as well.